### PR TITLE
Add in check_template option

### PR DESCRIPTION
**Pull request by ghostdevv**
Currently when using `svelte-kit package` on it's own it fails due to not finding a `app.html`, this pr makes the package command skip the check for app.html and makes it default for everything else

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

**Comment by ghostdevv** [comment]
Don't believe the netlify things are failing because of this pr

File: .changeset/famous-donuts-whisper.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+add in a check_template option to config loaders
```
Comment (bluwy) [inline .changeset/famous-donuts-whisper.md:5]: I think this should describe from the user point of view, e.g.
```suggestion
make html template optional for `svelte-kit package`
```

**Comment by Conduitry** [comment]
This would resolve #2547.

**Comment by ghostdevv** [comment]
Awesome, I know you mentioned in that issue that you might not want to encourage users to use svelte kit package standalone - for all my other libs I use it all together but for this particular project the documentation makes more sense being in Routify so I want to use svelte kit package essentially standalone

**Comment by Rich-Harris** [comment]
Thank you! Because of this I realised that we're actually doing the validation at the wrong time — in dev, it's possible to _start_ with a valid template, but then make it invalid without raising an error. What we really need to do is validate it at the point of use. I've added a `load_template` function that `dev` and `build` can both use — that way `load_config` doesn't need to know anything about template validity.

**Review by Rich-Harris — APPROVED** [review]

**Comment by ghostdevv** [comment]
Amazing thank you

**Pull request merged**